jsesc.js

```diff
@@ -95,6 +95,7 @@
 			'es6': false,
 			'json': false,
 			'compact': true,
+			'toUpperCase': true,
```

Comment:
I’m not sure `toUpperCase` is the best option name — how about `upperCaseHex` instead?